src/components/tableRow.jsx

```diff
@@ -8,6 +8,12 @@ import TableCell from "./tableCell";
 
 class TableRow extends Component<Object> {
 
+  constructor(props: Props) {
+    super(props);
+
+    this.handleRowClick = new handleRowClick();
```

Comment:
I don't understand why you did this.

Please change this to 
`this.handleRowClick = this.handleRowClick.bind(this)`

https://reactjs.org/docs/faq-functions.html#bind-in-constructor-es2015